src/editor/utils/domManipulate.js

```diff
@@ -170,6 +170,33 @@ export const createInputInCodeBlock = codeEle => {
   return inputInCodeFence
 }
 
+export const getTextContent = (node, blackList) => {
+  if (!blackList) return node.textContent
+  let text = ''
+  if (node.nodeType === 3) {
+    text += node.textContent
+  } else {
+    if (blackList.some(className => node.classList && node.classList.contains(className))) {
```

Comment:
I would change this to a one liner or use else if and put L182 to 184 into else.

Reply:
Sorry, I can not get your point, Can you write some code?

Reply:
if (node.nodeType === 3) {
    text += node.textContent
  } else {
    if (blackList.some(className => node.classList && node.classList.contains(className))) {

Reply:
👍